server/storage/database_test.go

```diff
@@ -31,6 +31,18 @@ func SampleTUF(version int) TUFFile {
 	}
 }
 
+func SampleCustomTUF(role, gun string, data []byte, version int) TUFFile {
```

Comment:
Totally non-blocking - wondering if it'd make sense `func SampleTuf` to make use of this function?